src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/OmniSharpTests.cs

```diff
@@ -18,6 +18,9 @@ namespace Microsoft.DotNet.SourceBuild.SmokeTests;
 /// </summary>
 public class OmniSharpTests : SmokeTests
 {
+    // Update version as new releases become available: https://github.com/OmniSharp/omnisharp-roslyn/releases
+    private const string OmniSharpReleaseVersion = "1.39.8";
```

Comment:
Is there anyway we could utilize dependabot to update this for us?

Reply:
Well that's unfortunate.  If the simple solutions are been exhausted, perhaps we should log a follow-up to automate this.  In the meantime would it make sense to add a step to the release checklist?

Reply:
Sure, I'll log a follow-up issue to automate. I can update the release checklist but that's a different repo.